tools/devtools/utils.mjs

```diff
@@ -0,0 +1,262 @@
+import {dirname} from 'path';
+
+const verholder = '$version';
+const langholder = '$lang';
+
+const blogTemplate = './tools/devtools/templates/new-in-devtools.md';
+const bannerTemplate = './tools/devtools/templates/new-in-devtools-banner.svg';
+
+const blogDest = `./site/${langholder}/blog/new-in-devtools-${verholder}/index.md`;
+const outlineDest = `./site/_includes/partials/devtools/${langholder}/whats-new.md`;
+const bannerDest = `./_temp/new-in-devtools-banner-${langholder}.svg`;
+
+const translation = {
+  language: {
+    es: 'Spanish',
+    ja: 'Japanese',
+    ko: 'Korean',
+    pt: 'Portuguese',
+    ru: 'Russian',
+    zh: 'Chinese',
+  },
+  title: {
+    en: "What's New in DevTools (Chrome $version)",
+    es: 'Qué hay de nuevo en DevTools (Chrome $version)',
+    ja: 'DevTools の新機能 (Chrome $version)',
+    ko: 'DevTools 의 새로운 기능 (Chrome $version)',
+    pt: 'O que há de novo no DevTools (Chrome $version)',
+    ru: 'Новинки DevTools (Chrome $version)',
+    zh: 'DevTools 新功能（Chrome $version）',
+  },
+  thankful: {
+    es: '*Gracias  por la traducción [Miguel Ángel](https://midu.dev) y por la revisión [Carlos Caballero](https://carloscaballero.io).*',
+    ja: '*翻訳者の [technohippy](https://github.com/technohippy) さん、レビュアーの [yoichiro](https://github.com/yoichiro) さん、 [lacolaco](https://github.com/lacolaco) さん、 [yoshiko-pg](https://github.com/yoshiko-pg) さんに感謝いたします。*',
+    ko: '*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하셨으며, [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님과 [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*',
+    pt: '*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/) . Revisão por [Lucas Santos](https://lsantos.dev).*',
+    ru: '*Переводы предоставлены [Alena Batitskaia](https://twitter.com/ABatickaya). Редактор — [Maxim Salnikov](https://twitter.com/webmaxru).*',
+    zh: '*感谢 [Poong Zui Yong](https://www.linkedin.com/in/zui-yong-poong-1b507b14/) 提供的翻译*',
+  },
+  reviewers: {
+    es: ['@Caballerog', '@midudev', '@and-oli'],
+    ja: ['@yoichiro', '@yoshiko-pg', '@technohippy', '@lacolaco'],
+    ko: ['@techhtml', '@TORU0239', '@cwdoh'],
+    pt: ['@khaosdoctor', '@alvarocamillont'],
+    ru: ['@solarrust', '@webmaxru', '@kateryna-prokopenko'],
+    zh: ['@louisyoong', '@liuliangsir', '@xyugroup', '@aquaMAX', '@hanselfmu'],
+  },
+  banner: {
+    en: "What's new in",
+    es: 'Qué hay de nuevo en',
+    ja: '新機能',
+    ko: '새로운 기능',
+    pt: 'O que há de novo no',
+    ru: 'Новинки',
+    zh: '新功能',
+  },
+};
+
+/**
+ * Get Today's date in ISO 8601
+ * @returns YYYY-MM-DD
+ */
+function getToday() {
+  const dateObj = new Date();
+  const year = dateObj.getUTCFullYear();
+  const month = (dateObj.getMonth() + 1).toString().padStart(2, '0');
+  const date = dateObj.getUTCDate().toString().padStart(2, '0');
+
+  return `${year}-${month}-${date}`;
+}
```

Comment:
It's a shame DateTimeFormat does not allow custom formats, but what you could do is:

`return new Intl.DateTimeFormat("fr-CA", {year: 'numeric', month: '2-digit', day: '2-digit'}).format(Date.now());`

That's a little hacky though as the `fr-CA` locale is what takes care of the order then.